Binary files wxWidgets.orig/buildMac/coredll_artmac.o and wxWidgets/buildMac/coredll_artmac.o differ
Binary files wxWidgets.orig/buildMac/coredll_carbon_icon.o and wxWidgets/buildMac/coredll_carbon_icon.o differ
Binary files wxWidgets.orig/buildMac/lib/libwx_osx_cocoau_core-3.1.0.0.0.dylib and wxWidgets/buildMac/lib/libwx_osx_cocoau_core-3.1.0.0.0.dylib differ
Binary files wxWidgets.orig/buildMac/lib/libwx_osx_cocoau_core-3.1.0.dylib and wxWidgets/buildMac/lib/libwx_osx_cocoau_core-3.1.0.dylib differ
Binary files wxWidgets.orig/buildMac/lib/libwx_osx_cocoau_core-3.1.dylib and wxWidgets/buildMac/lib/libwx_osx_cocoau_core-3.1.dylib differ
Only in wxWidgets/buildMac/samples/artprov: .deps
Only in wxWidgets/buildMac/samples/artprov: arttest
Only in wxWidgets/buildMac/samples/artprov: arttest.app
Only in wxWidgets/buildMac/samples/artprov: arttest_artbrows.o
Only in wxWidgets/buildMac/samples/artprov: arttest_arttest.o
diff -bru wxWidgets.orig/src/osx/artmac.cpp wxWidgets/src/osx/artmac.cpp
--- wxWidgets.orig/src/osx/artmac.cpp	2014-12-07 23:10:14.000000000 -0500
+++ wxWidgets/src/osx/artmac.cpp	2014-12-08 23:01:36.000000000 -0500
@@ -87,6 +87,7 @@
     ART_MSGBOX(wxART_FLOPPY,      wxICON_FLOPPY,      floppy)
     ART_MSGBOX(wxART_HARDDISK,    wxICON_HARDDISK,    harddisk)
     ART_MSGBOX(wxART_REMOVABLE,   wxICON_REMOVABLE,   removable)
+    ART_MSGBOX(wxART_PRINT,       wxICON_PRINT,       print)
 
     ART_MSGBOX(wxART_DELETE,      wxICON_DELETE,      delete)
 
@@ -96,6 +97,7 @@
 
     ART_MSGBOX(wxART_HELP_SETTINGS, wxICON_HELP_SETTINGS, htmoptns)
     ART_MSGBOX(wxART_HELP_PAGE,   wxICON_HELP_PAGE,   htmpage)
+    ART_MSGBOX(wxART_HELP_FOLDER,   wxICON_HELP_FOLDER,   htmlfoldr)
 
     return wxNullIconBundle;
 }
diff -bru wxWidgets.orig/src/osx/carbon/icon.cpp wxWidgets/src/osx/carbon/icon.cpp
--- wxWidgets.orig/src/osx/carbon/icon.cpp	2014-12-07 23:10:14.000000000 -0500
+++ wxWidgets/src/osx/carbon/icon.cpp	2014-12-08 23:03:52.000000000 -0500
@@ -316,6 +316,14 @@
     {
         theId = kGenericDocumentIcon ;
     }
+    else if( resourceName == wxT( "wxICON_PRINT" ) )
+    {
+        theId = kPrintMonitorFolderIcon;
+    }
+    else if( resourceName == wxT( "wxICON_HELP_FOLDER" ) )
+    {
+        theId = kHelpFolderIcon;
+    }
 
     if ( theId != 0 )
     {
